seacrowd/sea_datasets/unisent/unisent.py

```diff
@@ -0,0 +1,286 @@
+author={Asgari, Ehsaneddin and Braune, Fabienne and Ringlstetter, Christoph and Mofrad, Mohammad RK},
+booktitle={Proceedings of the International Conference on Language Resources and Evaluation (LREC-2020)},
+year={2020},
+organization={European Language Resources Association (ELRA)}
+}
+"""
+_DATASETNAME = "unisent"
+_DESCRIPTION = """\
+UniSent is a universal sentiment lexica for 1000+ languages.
+To build UniSent, the authors use a massively parallel Bible
+corpus to project sentiment information from English to other
+languages for sentiment analysis on Twitter data. 183 of 1404
+languages are spoken in Southeast Asia
+"""
+URL = "https://raw.githubusercontent.com/ehsanasgari/UniSent/master/unisent_lexica_v1/{}_unisent_lexicon.txt"
+_HOMEPAGE = "https://github.com/ehsanasgari/UniSent"
+_LANGUAGES = [
+    "aaz",
+    "abx",
+    "ace",
+    "agn",
+    "agt",
+    "ahk",
+    "akb",
+    "alj",
+    "alp",
+    "amk",
+    "aoz",
+    "atb",
+    "atd",
+    "att",
+    "ban",
+    "bbc",
+    "bcl",
+    "bgr",
+    "bgs",
+    "bgz",
+    "bhp",
+    "bkd",
+    "bku",
+    "blw",
+    "blz",
+    "bpr",
+    "bps",
+    "bru",
+    "btd",
+    "bth",
+    "bto",
+    "bts",
+    "btx",
+    "bug",
+    "bvz",
+    "bzi",
+    "cbk",
+    "ceb",
+    "cfm",
+    "cgc",
+    "clu",
+    "cmo",
+    "cnh",
+    "cnw",
+    "csy",
+    "ctd",
+    "czt",
+    "dgc",
+    "dtp",
+    "due",
+    "duo",
+    "ebk",
+    "fil",
+    "gbi",
+    "gor",
+    "heg",
+    "hil",
+    "hnj",
+    "hnn",
+    "hvn",
+    "iba",
+    "ifa",
+    "ifb",
+    "ifk",
+    "ifu",
+    "ify",
+    "ilo",
+    "ind",
+    "iry",
+    "isd",
+    "itv",
+    "ium",
+    "ivb",
+    "ivv",
+    "jav",
+    "jra",
+    "kac",
+    "khm",
+    "kix",
+    "kje",
+    "kmk",
+    "kne",
+    "kqe",
+    "krj",
+    "ksc",
+    "ksw",
+    "kxm",
+    "lao",
+    "lbk",
+    "lew",
+    "lex",
+    "lhi",
+    "lhu",
+    "ljp",
+    "lus",
+    "mad",
+    "mak",
+    "mbb",
+    "mbd",
+    "mbf",
+    "mbi",
+    "mbs",
+    "mbt",
+    "mej",
+    "mkn",
+    "mnb",
+    "mog",
+    "mqj",
+    "mqy",
+    "mrw",
+    "msb",
+    "msk",
+    "msm",
+    "mta",
+    "mtg",
+    "mtj",
+    "mvp",
+    "mwq",
+    "mwv",
+    "mya",
+    "nbe",
+    "nfa",
+    "nia",
+    "nij",
+    "nlc",
+    "npy",
+    "obo",
+    "pag",
+    "pam",
+    "plw",
+    "pmf",
+    "pne",
+    "ppk",
+    "prf",
+    "prk",
+    "ptu",
+    "pww",
+    "sas",
+    "sbl",
+    "sda",
+    "sgb",
+    "smk",
+    "sml",
+    "sun",
+    "sxn",
+    "szb",
+    "tbl",
+    "tby",
+    "tcz",
+    "tdt",
+    "tgl",
+    "tha",
+    "tih",
+    "tlb",
+    "twu",
+    "urk",
+    "vie",
+    "war",
+    "whk",
+    "wrs",
+    "xbr",
+    "yli",
+    "yva",
+    "zom",
+    "zyp",
+    "pse",
+    "mnx",
+    "mmn",
+    "lsi",
+    "hlt",
+    "gdg",
+    "bnj",
+    "acn",
+]
+
+_LICENSE = Licenses.CC_BY_NC_ND_4_0.value  # cc-by-nc-nd-4.0
+_LOCAL = False
+_URLS = {
```

Comment:
I'd rather to keep the actual `_URLS` variable as format-able string which can be adjusted to lang which the subset is referring to, in `_split_generators` (L258)